line-bot-cli/build.gradle.kts

```diff
@@ -24,24 +26,25 @@ dependencies {
     implementation("org.yaml:snakeyaml")
 }
 
-jar {
+tasks.jar {
     enabled = true
     archiveClassifier.set("")
 }
 
-bootJar {
+val bootJar = tasks.getByName<BootJar>("bootJar") {
```

Comment:
[nits] assignment to val `bootJar` is not essential here.

Reply:
yep. i'll care this later.